pkg/ccl/backupccl/restore_data_processor.go

```diff
@@ -188,6 +203,9 @@ func (rd *restoreDataProcessor) processRestoreSpanEntry(
 		iters = append(iters, iter)
 	}
 
+	timeOpeningIters := timeutil.Since(startTime)
+	rd.totalIterators += timeOpeningIters
+
 	batcher, err := bulk.MakeSSTBatcher(ctx, db, evalCtx.Settings,
```

Comment:
I wonder if we should move this to creation of the processor, so that we can get the aggregate timing over the whole processor lifetime.

Reply:
Moved this to the creation of the processor. I exposed the per-batch bulk op summary here. I still flush and reset on every entry though. I agree that it makes sense to move that out so that thee batcher can flush when it wants, but will address that in another PR since that change is more involved.